### PR TITLE
fix: adding `paste` shortcut tip to the start of each tutorial

### LINKED ISSUE #30453: About Paste command; and about the Ctrl + Alt + UpArrow/DownArrow shortcuts
In the Focus Mode, in the editor.

There is no Paste feature in the context menu when I right-click in the edit area. There is also no Paste command in the F1/Command palette. Because of both, I have to use Ctrl + V all the times. Please add the Paste feature.

The shortcuts <Ctrl + Alt + UpArrow> and <Ctrl + Alt + DownArrow> in the edit area do not work. Windows captures these shortcuts and flips the display instead. Please fix the functionality of these shortcuts.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 516a8fed-04ea-628c-d644-eb81dedecfe6
* Version Independent ID: 060c37e4-36e1-48b3-2435-cf858b14d860
* Content: [Data collections - Introduction to C# interactive tutorial](https://docs.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/tutorials/list-collection)
* Content Source: [docs/csharp/tour-of-csharp/tutorials/list-collection.yml](https://github.com/dotnet/docs/blob/main/docs/csharp/tour-of-csharp/tutorials/list-collection.yml)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by luizhlelis**
## Summary

Adding the `paste` shortcut tip to the start of each tutorial.

Fixes #30453

### CODE REVIEW

File: docs/csharp/tour-of-csharp/tutorials/branches-and-loops-local.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ ms.date: 03/23/2022
 
 This tutorial teaches you how to write C# code that examines variables and changes the execution path based on those variables. You write C# code and see the results of compiling and running it. The tutorial contains a series of lessons that explore branching and looping constructs in C#. These lessons teach you the fundamentals of the C# language.
 
+> Note: To paste a code snippet inside the `focus mode` you should use your keyboard shortcut (ctrl+v, or cmd+v).
```
Comment (IEvangelist) [inline docs/csharp/tour-of-csharp/tutorials/branches-and-loops-local.md:11]:
```suggestion
> [!TIP]
> To paste a code snippet inside the **focus mode** you should use your keyboard shortcut (<kbd>Ctrl</kbd> + <kbd>v</kbd>, or <kbd>cmd</kbd> + <kbd>v</kbd>).
```

File: docs/csharp/tour-of-csharp/tutorials/branches-and-loops.yml [hunk 2]
```diff
@@ -16,6 +16,8 @@ items:
 - durationInMinutes: 1
   content: |
     This tutorial teaches you how to write code that examines variables and changes execution path based on those variables. You'll use your browser to write C# interactively and see the results of compiling and running your code. This tutorial contains a series of lessons that explore branching and looping constructs in C#. These lessons teach you the fundamentals of the C# language.
+
+    > Note: To paste a code snippet inside the `focus mode` you should use your keyboard shortcut (ctrl+v, or cmd+v).
```
Comment (IEvangelist) [inline docs/csharp/tour-of-csharp/tutorials/branches-and-loops.yml:20]:
```suggestion
    > [!TIP]
    > To paste a code snippet inside the **focus mode** you should use your keyboard shortcut (<kbd>Ctrl</kbd> + <kbd>v</kbd>, or <kbd>cmd</kbd> + <kbd>v</kbd>).
```

File: docs/csharp/tour-of-csharp/tutorials/hello-world.yml [hunk 3]
```diff
@@ -16,6 +16,8 @@ items:
 - durationInMinutes: 1
   content: |
     This tutorial teaches you C# interactively, using your browser to write C# and see the results of compiling and running your code. It contains a series of lessons that begin with a "Hello World" program. These lessons teach you the fundamentals of the C# language.
+    
+    > Note: To paste a code snippet inside the `focus mode` you should use your keyboard shortcut (ctrl+v, or cmd+v).
```
Comment (IEvangelist) [inline docs/csharp/tour-of-csharp/tutorials/hello-world.yml:20]:
```suggestion
    > [!TIP]
    > To paste a code snippet inside the **focus mode** you should use your keyboard shortcut (<kbd>Ctrl</kbd> + <kbd>v</kbd>, or <kbd>cmd</kbd> + <kbd>v</kbd>).
```

File: docs/csharp/tour-of-csharp/tutorials/list-collection.yml [hunk 4]
```diff
@@ -14,6 +14,8 @@ items:
 - durationInMinutes: 1
   content: |
     This tutorial teaches you C# interactively, using your browser to write C# code and see the results of compiling and running your code. It contains a series of lessons that create, modify, and explore collections and arrays.
+    
+    > Note: To paste a code snippet inside the `focus mode` you should use your keyboard shortcut (ctrl+v, or cmd+v).
```
Comment (IEvangelist) [inline docs/csharp/tour-of-csharp/tutorials/list-collection.yml:18]:
```suggestion
    > [!TIP]
> To paste a code snippet inside the **focus mode** you should use your keyboard shortcut (<kbd>Ctrl</kbd> + <kbd>v</kbd>, or <kbd>cmd</kbd> + <kbd>v</kbd>).
```

File: docs/csharp/tour-of-csharp/tutorials/numbers-in-csharp-local.md [hunk 5]
```diff
@@ -8,6 +8,8 @@ ms.date: 03/23/2022
 
 This tutorial teaches you about the numeric types in C#. You'll write small amounts of code, then you'll compile and run that code. The tutorial contains a series of lessons that explore numbers and math operations in C#. These lessons teach you the fundamentals of the C# language.
 
+> Note: To paste a code snippet inside the `focus mode` you should use your keyboard shortcut (ctrl+v, or cmd+v).
```
Comment (IEvangelist) [inline docs/csharp/tour-of-csharp/tutorials/numbers-in-csharp-local.md:11]:
```suggestion
> [!TIP]
> To paste a code snippet inside the **focus mode** you should use your keyboard shortcut (<kbd>Ctrl</kbd> + <kbd>v</kbd>, or <kbd>cmd</kbd> + <kbd>v</kbd>).
```

File: docs/csharp/tour-of-csharp/tutorials/numbers-in-csharp.yml [hunk 6]
```diff
@@ -16,6 +16,8 @@ items:
 - durationInMinutes: 1
   content: |
     This tutorial teaches you about the numeric types in C# interactively, using your browser. You'll write C# and see the results of compiling and running your code. It contains a series of lessons that explore numbers and math operations in C#. These lessons teach you the fundamentals of the C# language.
+    
+    > Note: To paste a code snippet inside the `focus mode` you should use your keyboard shortcut (ctrl+v, or cmd+v).
```
Comment (IEvangelist) [inline docs/csharp/tour-of-csharp/tutorials/numbers-in-csharp.yml:20]:
```suggestion
    > [!TIP]
    > To paste a code snippet inside the **focus mode** you should use your keyboard shortcut (<kbd>Ctrl</kbd> + <kbd>v</kbd>, or <kbd>cmd</kbd> + <kbd>v</kbd>).
```

**Review by IEvangelist — COMMENTED** [review]
Let's try the [alert syntax](https://learn.microsoft.com/contribute/markdown-reference#alerts-note-tip-important-caution-warning) and use the `<kbd>` to correctly render keyboard shortcuts.

File: docs/csharp/tour-of-csharp/tutorials/branches-and-loops-local.md (hunk 1 above)
Comment (luizhlelis) [reply docs/csharp/tour-of-csharp/tutorials/branches-and-loops-local.md:11]: thank you @IEvangelist, I appreciate your review ✌🏽

File: docs/csharp/tour-of-csharp/tutorials/list-collection.yml [hunk 7]
```diff
@@ -14,6 +14,9 @@ items:
 - durationInMinutes: 1
   content: |
     This tutorial teaches you C# interactively, using your browser to write C# code and see the results of compiling and running your code. It contains a series of lessons that create, modify, and explore collections and arrays.
+    
+    > [!TIP]
+> To paste a code snippet inside the **focus mode** you should use your keyboard shortcut (<kbd>Ctrl</kbd> + <kbd>v</kbd>, or <kbd>cmd</kbd> + <kbd>v</kbd>).
```
Comment (IEvangelist) [inline docs/csharp/tour-of-csharp/tutorials/list-collection.yml:19]:
```suggestion
    > To paste a code snippet inside the **focus mode** you should use your keyboard shortcut (<kbd>Ctrl</kbd> + <kbd>v</kbd>, or <kbd>cmd</kbd> + <kbd>v</kbd>).
```

**Review by IEvangelist — APPROVED** [review]

**Pull request merged**